cw_bitcoin/lib/litecoin_wallet.dart

```diff
@@ -789,15 +833,23 @@ abstract class LitecoinWalletBase extends ElectrumWallet with Store {
     int unconfirmedMweb = 0;
     try {
       mwebUtxosBox.values.forEach((utxo) {
-        if (utxo.height > 0) {
+        bool isConfirmed = utxo.height > 0;
+
+        print(
+            "utxo: ${isConfirmed ? "confirmed" : "unconfirmed"} ${utxo.spent ? "spent" : "unspent"} ${utxo.outputId} ${utxo.height} ${utxo.value}");
+
+        if (isConfirmed) {
           confirmedMweb += utxo.value.toInt();
-        } else {
+        }
+
+        if (isConfirmed && utxo.spent) {
+          unconfirmedMweb -= utxo.value.toInt();
```

Comment:
er, the original if condition might be correct I'll need to do some more testing

Reply:
what did you find regarding this?

Reply:
so this is here so that once we try to spend some coins, they briefly show up in both the confirmed balance, and the unconfirmed balance as negative

i.e. if I have 1 ltc and then spend it, I still have 1 ltc (confirmed) and -1 ltc (unconfirmed)
once the transaction gets a confirmation we no longer count the utxo at all (it is deleted) and your balance would become 0 confirmed, 0 unconfirmed

Reply:
basically this is here for consistency with how regular ltc balances are displayed